client/state/jetpack-settings/modules/test/actions.js

```diff
@@ -47,7 +46,17 @@ describe( 'actions', () => {
 			useNock( ( nock ) => {
 				nock( 'https://public-api.wordpress.com:443' )
 				.post( '/rest/v1.1/sites/123456/jetpack/modules/module-a' )
-				.reply( 200, {} );
+				.reply( 200, {
```

Comment:
Can we have this object in the `fixture.js` file ? It can come in handy for a few other operations.